src/pages/TCY/queries/useTcyClaims.tsx

```diff
@@ -45,6 +48,9 @@ export const useTCYClaims = (accountNumber: number | 'all') => {
       queryKey: ['tcy-claims', accountId, isConnected],
       queryFn: async (): Promise<Claim[]> => {
         if (!isConnected) return []
+        const isMetaMaskMultichainWallet = wallet instanceof MetaMaskMultiChainHDWallet
+
+        if (isMetaMaskMultichainWallet && !isSnapInstalled) return []
```

Comment:
blocking: that won't invalidate on `isSnapInstalled` change, is that intended?

Reply:
Daaamn, this is some dusts, it was only crashing because MM was keeping unsupported assets accounts after uninstalling snap, reverted in [9baf06b](https://github.com/shapeshift/web/pull/9993/commits/9baf06baa189dbcb5654d6ecd28306dd9127edbb)